cypress/e2e/Tests/ViewAndCartBrandProducts.cy.js

```diff
@@ -0,0 +1,44 @@
+import homePage from "../PageObjects/HomePage";
+import ProductDetailsPage from "../PageObjects/ProductDetailsPage";
```

Comment:
I would change _ ProductDetailsPage_ to _ productDetailsPage_

Reply:
Fixed! 🙌🏼